### PR TITLE
Test for SET @@SESSION.sql_log_bin in sql static validations

### PR DESCRIPTION
**Pull request by elazzabi**
## Description

This adds a test for `SET @@SESSION.sql_log_bin` usually added by `mysqldump` to SQL files. More info https://dev.mysql.com/doc/refman/8.0/en/set-sql-log-bin.html

## Steps to Test

An automated test case was added.

### CODE REVIEW

File: src/lib/validations/sql.js [hunk 1]
```diff
@@ -118,6 +118,15 @@ const checks: Checks = {
 		excerpt: '\'CREATE DATABASE\' statement should not  be present (case-insensitive)',
 		recommendation: 'Remove these lines',
 	},
+	binaryLogging: {
+		matcher: /SET @@SESSION.sql_log_bin/i,
+		matchHandler: lineNumber => lineNumber,
+		outputFormatter: errorCheckFormatter,
+		results: [],
+		message: 'SET @@SESSION.sql_log_bin statement',
+		excerpt: '\'SET @@SESSION.sql_log_bin=0\' statement should not be present (case-insensitive)',
```
Comment (dchymko) [inline src/lib/validations/sql.js]: Super minor, but we probably don't need the `=0` here.

**Review by dchymko — DISMISSED** [review]
LGTM!

File: src/lib/validations/sql.js (hunk 1 above)
Comment (elazzabi) [reply src/lib/validations/sql.js]: Oh yeah good catch!

Comment (elazzabi) [reply src/lib/validations/sql.js]: Fixed!

**Comment by elazzabi** [comment]
@markowsiak @jblz forgot to merge this before, so going to merge it now.

**Pull request merged**